waspc/examples/todoApp/src/server/webSocket.ts

```diff
@@ -0,0 +1,29 @@
+import { v4 as uuidv4 } from 'uuid'
+import { WebSocketDefinition } from '@wasp/webSocket'
+
+export interface ServerToClientEvents {
+  chatMessage: (msg: { id: string; username: string; text: string }) => void
+}
+
+export interface ClientToServerEvents {
+  chatMessage: (msg: string) => void
+}
+
+export interface InterServerEvents {}
```

Comment:
Why are we exporting these types? Does making them private break full-stack type safety? I've removed the exports and everything seems to be working fine.